conf/cassandra.yaml

```diff
@@ -1111,6 +1111,13 @@ compaction_large_partition_warning_threshold_mb: 100
 # as corrupted. This should be positive and less than 2048.
 # max_value_size_in_mb: 256
 
+# If replication factor is not mentioned as part of keyspace creation, default_keyspace_rf would apply.
+# This would also apply for any system keyspaces that need replication factor.
+#default_keyspace_rf: 1
+
+# Indicates the minimum replication factor that a keyspace should have.
+#minimum_keyspace_rf: 0
```

Comment:
While maintaining `minimum_keyspace_rf` is a good thing to do, but thinking about number options in yaml, was wondering if there is a better way to do the same. I see that `minimum_keyspace_rf` is only used in validations? so adding a configuration for validations seems to be overkill. Can we assume the default `minimum_keyspace_rf` or pass it via jvm argument if it can't be avoided?

Reply:
we would only need to define minimum_keyspace_rf if we are to override the default setting of 0 (which is same as today for backward compatibility reasons). From the view of ease of operation/maintenance, I thought yaml would be a good place to override this setting. Also, since setting this value can have a profound effect on keyspace creation, its good to be able to see what the value is, from yaml, from an operations perspective.

Reply:
Also nit: make sure to have spaces after comments (styleguide)

Reply:
`Also, since setting this value can have a profound effect on keyspace creation, its good to be able to see what the value is, from yaml, from an operations perspective.` 
Agreed on this point. But we might see pushback in the community for adding too many configs if the community thinks these are too many. So just keep it in your backburner to see if there is an alternative, we can at least get it rolling internally this way and iterate when we get feedback from community.